tyk-stack/README.md

```diff
@@ -258,158 +440,173 @@ To set up other backends for pump, refer to this [document](https://github.com/T
+#### Storage Settings
 
-#### Redis Password
-
-Redis password can also be provided via a secret. Store Redis password in Kubernetes secret and refer to this secret
-via `global.redis.passSecret.name` and `global.redis.passSecret.keyName` field, as follows:
+Tyk Enterprise Portal supports different storage options for storing the portal's CMS assets such as images, theme files and Open API Specification files. Please see the [Enterprise Portal Storage settings]({{<ref "tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/configuration#portal-settings">}}) page for all the available options. Helm chart supports the setting of following fields in values.yaml `storage` section:
 
 ```yaml
-global:  
-  redis:
-     passSecret:
-       name: "yourSecret"
-       keyName: "redisPassKey"
+tyk-dev-portal:
+  storage:
+    # Configuration values for using an SQL database as storage for Tyk Developer Portal
+    # In case you want to provide the connection string via secrets please
+    # refer to the existing secret inside the helmchart or the
+    # .Values.global.secrets.useSecretName variable
+    # User can set the storage type for portal.
+    # Supported types: fs, s3, db
+    type: "fs"
+    database:
+      dialect: "sqlite3"
+      connectionString: "db/portal.db"
+      enableLogs: false
+      maxRetries: 3
+      retryDelay: 5000
+    # Configuration values for using s3 as storage for Tyk Developer Portal
+    # In case you want to provide the key ID and access key via secrets please
+    # refer to the existing secret inside the helmchart or the
+    # .Values.global.secrets.useSecretName variable and a secret containing
+    # the keys DeveloperPortalAwsAccessKeyId and respectively,
+    # DeveloperPortalAwsSecretAccessKey
+    s3:
+      awsAccessKeyid: your-access-key
+      awsSecretAccessKey: your-secret-key
+      region: sa-east-1
+      endpoint: your-portal-bucket
+      bucket: https://s3.sa-east-1.amazonaws.com
+      acl: private
+      presign_urls: true
+    persistence:
+      # User can mount existing PVC to the Tyk Developer Portal
+      # Make sure to change the kind to Deployment if you are mounting existing PVC 
+      mountExistingPVC: ""
+      storageClass: ""
+      accessModes:
+        - ReadWriteOnce
+      size: 8Gi
+      annotations: {}
+      labels: {}
+      selector: {}
 ```
 
-#### MongoDB or Postgres connection strings
+#### Other Configurations
 
-Storage connection strings can also be provided via a secret. Store the connection string in Kubernetes secret and 
-refer to this secret via `global.{mongo,postgres}.connectionURLSecret.name` and `global.{mongo,postgres}.connectionURLSecret.keyName` field, 
-as follows:
+Other [Enterprise Portal configurations]({{<ref "tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/configuration">}}) can be set by using environment variables with `extraEnvs` fields, e.g.:
```

Comment:
This linking `{{<ref "tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/configuration">}}` won't work in GitHub readme.

Reply:
```suggestion
Other [Enterprise Portal configurations](https://tyk.io/docs/tyk-stack/tyk-developer-portal/enterprise-developer-portal/install-tyk-enterprise-portal/configuration) can be set by using environment variables with `extraEnvs` fields, e.g.:
```